WcaOnRails/Gemfile

```diff
@@ -45,6 +45,7 @@ gem 'octokit'
 gem 'stripe'
 gem 'oauth2'
 gem 'openssl'
+gem 'httparty'
```

Comment:
I'm quite sure we should already have something for POST requests in our zoo of dependencies.

Also, why is `Gemfile.lock` not part of the diff?

Reply:
because I didn't actually install it locally :P. I will check the other dependencies, but I don't think we do

Reply:
If you want this gem, you definitely have to install it and check in the changes to `Gemfile.lock`.

Reply:
Also, there is [`faraday`](https://github.com/lostisland/faraday) already in our Gemfile, which I think does what you want.